internal/command/command.go

```diff
@@ -78,15 +86,15 @@ func (c Client) branchName() (string, error) {
 	return ret, nil
 }
 
-func (c Client) revParse(branch string) (string, error) {
+func (c Client) headSha() (string, error) {
 	/* #nosec */
-	cmd := exec.Command("git", "-C", c.Workspace, "rev-parse", branch)
+	cmd := exec.Command("git", "-C", c.Workspace, "rev-parse", "HEAD")
```

Comment:
I made this change because we no longer need the ability to get the sha of arbitrary branches